src/Product.php

```diff
@@ -308,6 +308,10 @@ public function setField($key, $value, $array = false)
      */
     public function setAttributeField($name, $value)
     {
+        if ($name === null || $value === null) {
```

Comment:
This is a pretty strict null check, should we use `isset()` instead?

Reply:
How would such a check look like? I want a strict check because I have a strict type check inside of my method a few lines below.

Reply:
`if (!isset($name) || !isset($value))`

Reply:
As discussed in person, it would be the same result, therefore skipped.